src/AudioFileSourceHTTPStream.cpp

```diff
@@ -24,16 +24,58 @@
 
 AudioFileSourceHTTPStream::AudioFileSourceHTTPStream()
 {
+ // init_ascii_to_hex();
   pos = 0;
   reconnectTries = 0;
   saveURL[0] = 0;
+  next_chunk = 0;
 }
 
 AudioFileSourceHTTPStream::AudioFileSourceHTTPStream(const char *url)
 {
+  //init_ascii_to_hex();
   saveURL[0] = 0;
   reconnectTries = 0;
+  next_chunk = 0;
   open(url);
+
+}
+
+int AudioFileSourceHTTPStream::convertHexToInt(const char *str)
+{
+  int result = 0;
+  while(*str)
+  {
+    if(!((*str <= '9' && *str >= '0') || (*str <= 'f' && *str >= 'a')))
+    {
+      audioLogger->printf("Character not HEX");
+      return -1;
+    }
+    result = (result << 4) | ascii_to_hex[*str];
+    ++str; 
+  }
+
+  return result;
+}
+
+bool AudioFileSourceHTTPStream::verifyCrlf()
+{
+  
+  uint8_t crlf[3];
+  
+  client.read(crlf, 2);
+  crlf[2] = 0;
+  
+  String crlfString = "\r\n";
```

Comment:
Arduino Strings are kind of heavyweight.  Why not `return !strncmp(crlf, "\r\n", 2);`?

Reply:
Fixed !